packages/teams-js/src/public/pages.ts

```diff
@@ -372,28 +371,28 @@ export namespace pages {
    */
   export interface NavigateToAppParams {
     /**
-     * ID of the application to navigate to
+     * ID of the app to navigate to
      */
     appId: string;
 
     /**
-     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
+     * Developer-defined ID of the page to navigate to within the app (formerly called `entityId`)
      */
     pageId: string;
 
     /**
-     * Optional URL to open if the navigation cannot be completed within the host
+     * Fallback URL to open if the navigation cannot be completed within the host (e.g. if the target app is not installed)
      */
     webUrl?: string;
 
     /**
-     * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
-     * back to the application via the Context object.
+     * Developer-defined ID describing the content to navigate to within the page. This will be passed to the application
```

Comment:
Just out of curiosity, what was the motivation behind removing the note that it was optional?

Reply:
It seemed duplicative with the fact that the property is marked with a `?`.